sublime_linter.py

```diff
@@ -332,22 +316,25 @@ def join_msgs(cls, line_dict, we_count, show_count=False):
                 <div>{messages}</div>
             '''
 
-        template = "{linter}: {code} - {escaped_msg}"
-        template_no_code = "{linter}: {escaped_msg}"
+        tmpl_with_code = "{linter}: {code} - {escaped_msg}"
+        tmpl_sans_code = "{linter}: {escaped_msg}"
 
         all_msgs = ""
         for error_type in WARN_ERR:
```

Comment:
The nested for-loop is probably not necessary.just `for error in errors`?

Reply:
We need to separate error types here. Do you see a cleaner solution than iterating over `("warning", "error")`?